### PR TITLE
PILOT-1769 Configure logger levels

### PR DESCRIPTION
**Pull request by erikdvlp**
## Summary

- Update common package to `0.1.0`.
- Add variables to the app config to control logger levels.
- Set all logger levels to 30 (warning) except for the error handler at 40 (error).

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1769

## Type of Change

- [x] New feature (non-breaking change which adds functionality)

## Testing

Are there any new or updated tests to validate the changes?

- [x] No

## Test Directions

N/A

### CODE REVIEW

**Comment by erikdvlp** [comment]
@hugoantunes Please also let me know if the health check logging spam is fixed either by this or [your previous work](https://indocconsortium.atlassian.net/browse/PILOT-1753) or not as I think they are related.

File: app/config.py [hunk 1]
```diff
@@ -59,9 +59,13 @@ class Settings(BaseSettings):
     MAX_TAGS = 10
     MAX_SYSTEM_TAGS = 10
     MAX_ATTRIBUTE_LENGTH = 100
-
     MAX_COLLECTIONS = 10
 
+    LOG_LEVEL_DEFAULT = 30
+    LOG_LEVEL_FILE = 30
+    LOG_LEVEL_STDOUT = 30
+    LOG_LEVEL_STDERR = 40
```
Comment (hugoantunes) [inline app/config.py:67]: can we change to more understandable values?
https://docs.python.org/3/library/logging.html#logging-levels

Comment (hugoantunes) [inline app/config.py:67]: Can you change it to more understandable values? 
https://docs.python.org/3/library/logging.html#logging-levels

**Review by hugoantunes — CHANGES_REQUESTED** [review]

**Comment by hugoantunes** [comment]
@erikdvlp the other PR fixed the logging spam.

**Comment by hugoantunes** [comment]
> @hugoantunes Please also let me know if the health check logging spam is fixed either by this or [your previous work](https://indocconsortium.atlassian.net/browse/PILOT-1753) or not as I think they are related.


@erikdvlp the other PR fixed the logging spam.

File: app/config.py (hunk 1 above)
Comment (erikdvlp) [reply app/config.py]: Done.

**Review by hugoantunes — APPROVED** [review]

**Pull request merged**